site/content/en/contribute/features/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Contributing Features"
+linkTitle: "Contributing Features"
+type: docs
+weight: 30
+description: >
+    How to contribute features
+---
+
+[issue]: https://github.com/kubernetes-sigs/kustomize/issues
+[sig-cli]: https://groups.google.com/g/kubernetes-sig-cli
+[meeting agenda]: https://docs.google.com/document/d/1r0YElcXt6G5mOWxwZiXgGu_X6he3F--wKwg-9UBc29I/edit#heading=h.himo1st0tqyy
+[KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli
+[table-driven]: https://github.com/kubernetes-sigs/kustomize/blob/master/api/filters/imagetag/imagetag_test.go#L16
+[eschewed feature list]: /faq/eschewedfeatures/
+[kind/feature]: https://github.com/kubernetes-sigs/kustomize/labels/kind%2Ffeature
+[sig-cli bi-weekly]: https://github.com/kubernetes/community/tree/master/sig-cli#meetings
+
+Following is the process for proposing a new Kustomize feature:
```

Comment:
oh, okie! makes sense!

I think just adding to the docs you shared makes sense since it already outlines what one should do. And yeah, I went through the issue template. It adds the label and everything.

So opting to just point to the proposals doc. Addressed here: https://github.com/kubernetes-sigs/kustomize/pull/4709/commits/361cf72a90c2b79bce3652c2067da1b401731272